torchmetrics/functional/text/wer.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any
+
+from jiwer import compute_measures
```

Comment:
we need to define a variable similar to these for `jiwer`:
https://github.com/PyTorchLightning/metrics/blob/5c2069eb2087ede7f008222ebab286aedad14dfb/torchmetrics/utilities/imports.py#L71-L78
such that people that do not have `jiwer` install can still use torchmetrics (just not this metric)

Reply:
Done